src/kernel/arch/x86/paging.zig

```diff
@@ -318,18 +327,23 @@ fn mapTableEntry(entry: *align(1) TableEntry, phys_addr: usize, attrs: vmm.Attri
     } else {
         setAttribute(entry, TENTRY_USER);
     }
-    if (attrs.writable) {
-        setAttribute(entry, TENTRY_WRITE_THROUGH);
-    } else {
-        clearAttribute(entry, TENTRY_WRITE_THROUGH);
-    }
+
+    clearAttribute(entry, TENTRY_WRITE_THROUGH);
+
```

Comment:
Looks like I misunderstood what write-through meant when I first wrote the paging code. When set, it means that changes are written directly to memory instead of being cached and written at a later time, so this should depend on the `cachable` attribute instead.